utils.ts

```diff
@@ -1,166 +1,243 @@
-import { parse } from 'lua-json'
-import { writeFileSync } from 'fs'
-import { join } from 'path'
-import { jobs, skills, slots } from './src/constants.json'
+import { parse } from "lua-json";
+import { writeFileSync } from "fs";
+import { join } from "path";
+import { jobs, skills, slots } from "./src/constants.json";
 
 export interface RawItem {
-    id: number
-    en: string
-    ja: string
-    enl: string
-    jal: string
-    category: Category
-    flags?: number
-    stack?: number
-    targets?: number
-    type?: number
-    cast_time?: number
-    jobs?: number
-    level?: number
-    races?: number
-    slots?: number
-    cast_delay?: number
-    max_charges?: number
-    recast_delay?: number
-    shield_size?: number
-    damage?: number
-    delay?: number
-    skill?: number
-    ammo_type?: AmmoType
-    range_type?: RangeType
-    item_level?: number
-    superior_level?: number
+  id: number;
+  en: string;
+  ja: string;
+  enl: string;
+  jal: string;
+  category: Category;
+  flags?: number;
+  stack?: number;
+  targets?: number;
+  type?: number;
+  cast_time?: number;
+  jobs?: number;
+  level?: number;
+  races?: number;
+  slots?: number;
+  cast_delay?: number;
+  max_charges?: number;
+  recast_delay?: number;
+  shield_size?: number;
+  damage?: number;
+  delay?: number;
+  skill?: number;
+  ammo_type?: AmmoType;
+  range_type?: RangeType;
+  item_level?: number;
+  superior_level?: number;
 }
 export enum AmmoType {
-    Arrow = "Arrow",
-    Bait = "Bait",
-    Bolt = "Bolt",
-    Bullet = "Bullet",
-    Shell = "Shell",
+  Arrow = "Arrow",
+  Bait = "Bait",
+  Bolt = "Bolt",
+  Bullet = "Bullet",
+  Shell = "Shell"
 }
 export enum Category {
-    Armor = "Armor",
-    Automaton = "Automaton",
-    General = "General",
-    Gil = "Gil",
-    Maze = "Maze",
-    Usable = "Usable",
-    Weapon = "Weapon",
+  Armor = "Armor",
+  Automaton = "Automaton",
+  General = "General",
+  Gil = "Gil",
+  Maze = "Maze",
+  Usable = "Usable",
+  Weapon = "Weapon"
 }
 export enum SkillCategory {
-    Combat = "Combat",
-    Magic = "Magic",
-    None = "None",
-    Puppet = "Puppet",
-    Synthesis = "Synthesis",
+  Combat = "Combat",
+  Magic = "Magic",
+  None = "None",
+  Puppet = "Puppet",
+  Synthesis = "Synthesis"
 }
 export enum RangeType {
-    Bow = "Bow",
-    Cannon = "Cannon",
-    Crossbow = "Crossbow",
-    FishingRod = "Fishing Rod",
-    Gun = "Gun",
+  Bow = "Bow",
+  Cannon = "Cannon",
+  Crossbow = "Crossbow",
+  FishingRod = "Fishing Rod",
+  Gun = "Gun"
 }
 export interface Item {
-    id: number
-    name: string
-    description: string
-    level: number
-    item_level: number
-    jobs: string
-    _jobs: number
-    _slots: number
-    skill: number
-    type: string
+  id: number;
+  name: string;
+  description: string;
+  level: number;
+  item_level: number;
+  jobs: string;
+  _jobs: number;
+  _slots: number;
+  skill: number;
+  type: string;
 }
 
+const AllJobs = 8388606;
```

Comment:
こう書くとわかりやすいですかねー？どうでしょう

```suggestion
const AllJobs = parseInt('11111111111111111111110', 2);
```